packages/@webex/plugin-meetings/src/rtcMetrics/index.ts

```diff
@@ -1,5 +1,26 @@
+import anonymizeIP from 'ip-anonymize';
 import RTC_METRICS from './constants';
 
+/**
+ * Anonymize identifiable information. For now, only IP addresses.
+ *
+ * @param {array} stats - An RTCStatsReport organized into an array of strings.
+ * @returns {void}
+ */
+const removePii = (stats: string[]): string[] => {
+  return stats.map((statLine) => {
+    const data = JSON.parse(statLine);
+    // on local and remote candidates, anonymize the last 4 bits.
+    if (data.type === 'local-candidate' || data.type === 'remote-candidate') {
+      data.ip = anonymizeIP(data.ip, 28, 96) || undefined;
```

Comment:
Can we not use the same function imported from call-diagnostic metrics util?

can do it like this: https://github.com/webex/webex-js-sdk/pull/3282/files

that way the code is centralized (anonymization)

Reply:
Updated.